isa-parser/Makefile

```diff
@@ -14,17 +14,20 @@ export E Q
 PWD:=$(shell pwd)
 include $(PWD)/../config.mk
 
-install:
+install: deps
 	$(E) "Installing.."
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(INSTALL_FLAGS)
 
-pinstall: CABALFLAGS += --enable-executable-profiling
+pinstall: CONFIG_FLAGS += --enable-executable-profiling
 pinstall: setup-config
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(CONFIG_FLAGS)
+
+deps:
+	$(CABAL) install --only-dependencies
```

Comment:
So that only leaves us with only one option - to get rid of the 'global install' option.